.jshintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "bitwise": true,
+  "curly": true,
+  "eqeqeq": true,
+  "freeze": true,
+  "futurehostile": true,
+  "latedef": "nofunc",
+  "noarg": true,
+  "nonew": true,
+  "shadow": false,
+  "strict": false,
+  "undef": true,
+  "globals": {
+    "access": false,
+    "channel": false,
+    "requireAccess": false,
+    "requireRole": false,
+    "requireUser": false
+  }
```

Comment:
You may notice that I switched out the use of `predefined` for `globals`. This allows me to control whether the value of each of the variables can be overwritten. A setting of `false` means the global variable in question can only be read and `true` means that it allows read and write. Note that each of these is overridden with a value of `true` in the `etc` and `test` directories' .jshintrc files because some test cases and the test helper need to be able to reassign the `channel` and `requireAccess` global variables, for example.